pipeline/transformer/transformer_miro/src/test/scala/weco/pipeline/transformer/miro/transformers/MiroContributorsTest.scala

```diff
@@ -20,68 +21,61 @@ class MiroRecordTransformerContributorsTest
   }
 
   it("passes through a single value in the image_creator field") {
-    val creator = "Researcher Rosie"
     transformRecordAndCheckContributors(
       miroRecord = createMiroRecordWith(
-        creator = Some(List(Some(creator)))
+        creator = Some(List(Some("Researcher Rosie")))
       ),
-      expectedContributors = List(creator)
+      expectedContributors = List(
+        ExpectedContributor(label = "Researcher Rosie", idLabel = "researcher rosie")
+      )
     )
   }
 
   it("ignores null values in the image_creator field") {
-    val creator1 = "Beekeeper Brian"
-    val creator2 = "Dog-owner Derek"
     transformRecordAndCheckContributors(
       miroRecord = createMiroRecordWith(
-        creator = Some(List(Some(creator1), None, Some(creator2)))
+        creator = Some(List(Some("Beekeeper Brian"), None, Some("Dog-owner Derek")))
       ),
-      expectedContributors = List(creator1, creator2)
-    )
-  }
-
-  it("passes through multiple values in the image_creator field") {
```

Comment:
I remove this test because it felt superfluous with the `ignores null values` test.